docs/modules/airflow/examples/getting_started/code/getting_started.sh.j2

```diff
@@ -18,20 +18,20 @@ then
 fi
 
 echo "Adding bitnami Helm Chart repository and dependencies (Postgresql and Redis)"
-# tag::helm-add-bitnami-repo[]
-helm repo add bitnami https://charts.bitnami.com/bitnami
-# end::helm-add-bitnami-repo[]
 # tag::helm-add-bitnami-pgs[]
-helm install --wait airflow-postgresql bitnami/postgresql --version {{ versions.postgresql }} \
-    --set auth.username=airflow \
-    --set auth.password=airflow \
-    --set auth.database=airflow
+helm install airflow-postgresql oci://registry-1.docker.io/bitnamicharts/postgresql \
+  --version {{ versions.postgresql }} \
+  --set auth.database=airflow \
+  --set auth.username=airflow \
+  --set auth.password=airflow \
+  --wait
 # end::helm-add-bitnami-pgs[]
 # tag::helm-add-bitnami-redis[]
-helm install --wait airflow-redis bitnami/redis \
-    --set auth.password=redis \
-    --version {{ versions.redis }} \
-    --set replica.replicaCount=1
+helm install airflow-postgresql oci://registry-1.docker.io/bitnamicharts/redis \
```

Comment:
```suggestion
helm install airflow-redis oci://registry-1.docker.io/bitnamicharts/redis \
```

Reply:
Fixed in e372f123904acb40cdd255526d9fe5be8faa4c53